app/build.gradle

```diff
@@ -115,6 +116,8 @@ dependencies {
     implementation 'tech.relaycorp:relaynet:1.36.3'
     implementation 'tech.relaycorp:relaynet-cogrpc:1.1.2'
     implementation 'tech.relaycorp:relaynet-cogrpc-okhttp:1.1.0'
+    testImplementation "tech.relaycorp:relaynet-testing:$relaynetTestingVersion"
+    androidTestImplementation "tech.relaycorp:relaynet-testing:$relaynetTestingVersion"
```

Comment:
Is there a way to specify this once? I couldn't find one.

Reply:
Nop, you need to specify it both. But I'm usually relegating the test dependencies to the bottom of the `dependencies` list.

Reply:
Good point! I'll keep in mind in the future for consistency